app.js

```diff
@@ -68,24 +76,38 @@ finishGameButton.addEventListener('click', async () => {
     displayCurrentGameEl();
 });
 
-// on load . . .
+
 window.addEventListener('', async () => {
-    // display all past games (hint: call displayAllGames())
+
+    displayAllGames();
+    
 });
 
 function displayCurrentGameEl() {
-    // clear out the current game div
-    // change the label to show team one's name;
-    // change the label to show team two's name;
-    // call the render game function to create a game element
-    // append the element to the cleared out current game div
+    currentGameEl.textContent = '';
+
+    const gameEl = renderGame({ name1, name2, score1, score2 });
+ 
+    currentGameEl.append(gameEl);
 }
 
-function displayAllGames() {
-    // clear out the past games list in the DOM
-    // FETCH ALL GAMES from supabase
+async function displayAllGames() {
+
+    const pastGames = await getGames();
+    pastGamesEl.textContent = '';
+   
+
     // loop through the past games
-    // render and append a past game for each past game in state
+    for (let game of pastGames) {
+        
+        const gameEl = renderGame (game.name1, game.name2, game.score1, game.score2);
```

Comment:
Look at the definition for `renderGame()`. How many parameters does it have vs. how many arguments are you giving it here?

You're also calling the `renderGame()` function inside the `displayCurrentGameEl()` function above. Can you spot the difference?